app/controllers/analysis_configurations_controller.rb

```diff
@@ -1,7 +1,6 @@
 class AnalysisConfigurationsController < ApplicationController
   before_action :set_analysis_configuration, only: [:show, :edit, :update, :destroy, :reset_analysis_parameters,
-                                                    :submission_preview, :load_study_for_submission_preview,
-                                                    :update_analysis_parameter]
```

Comment:
This is a regression - we lost the :update_analysis_parameter method.  I think this was probably just an accidental keystroke in the wrong file.

Reply:
After checking master we found that update_analysis_parameter is in fact suppose to be there